tests/robot/4_check_edi.robot

```diff
@@ -1,10 +1,12 @@
 *** Settings ***
 Documentation       Legion's EDI operational check
+Test Timeout        5 minutes
```

Comment:
Is 5 minutes enough?

Reply:
@kirillmakhonin more than enough, will configure more closely when it starts work.